packages/block-library/src/embed/embed-preview.js

```diff
@@ -55,7 +55,7 @@ const EmbedPreview = ( props ) => {
 			{ ( cannotPreview ) ? (
 				<Placeholder icon={ <BlockIcon icon={ icon } showColors /> } label={ label }>
 					<p className="components-placeholder__error"><a href={ url }>{ url }</a></p>
-					<p className="components-placeholder__error">{ __( 'Sorry, this embedded content cannot be previewed in the editor.' ) }</p>
+					<p className="components-placeholder__error">{ sprintf( __( "Embedded content from %s can't be previewed in the editor." ), parsedHostBaseUrl ) }</p>
```

Comment:
Note that this is problematic from an internationalisation (i18n) standpoint, since a number of languages expect the piece being interpolated (`%s`) to be inflected to reflect its syntactic function. Very synthetic languages like Finnish or Hungarian, just to name a few, are especially prone to this. See the same argument in 
https://github.com/WordPress/gutenberg/pull/6124#issuecomment-382443638.

That said, in this case it seems like the piece to interpolate is limited to domain names, so I don't think we need to be too strict. I do however suggest adding a `translators` comment above it:

```js
/* translators: %s: Domain name such as `example.org` */
<p className="components-placeholder__error">{ sprintf( __( "Embedded content from %s can't be previewed in the editor." ), parsedHostBaseUrl ) }</p>
```